src/python/turicreate/visualization/_plot.py

```diff
@@ -450,6 +450,7 @@ def image_formatter(im):
                             </style>                               \
                           </head>                                  \
                           <body>                                   \
+                            <h1> '+ title +' </h1>                 \
```

Comment:
`title` needs to be HTML encoded; otherwise some values of `title` could cause malformed HTML. You can use the [HTML.escape](https://docs.python.org/3/library/html.html#html.escape) in Python 3 and [cgi.escape](https://docs.python.org/3.6/library/cgi.html#cgi.escape) in Python 2 for this.

Reply:
Thanks @znation !  Will check and raise PR